examples/chip-tool/config/PersistentStorage.cpp

```diff
@@ -33,12 +33,12 @@ using namespace ::chip::Controller;
 using namespace ::chip::Logging;
 
 constexpr const char kFilename[]           = "/tmp/chip_tool_config.ini";
-constexpr const char kDefaultSectionName[] = "Default";
 constexpr const char kPortKey[]            = "ListenPort";
 constexpr const char kLoggingKey[]         = "LoggingLevel";
 constexpr const char kLocalNodeIdKey[]     = "LocalNodeId";
 constexpr const char kRemoteNodeIdKey[]    = "RemoteNodeId";
 constexpr LogCategory kDefaultLoggingLevel = kLogCategory_Detail;
+char kSectionName[32]                      = { 0 };
```

Comment:
this is not a constant, name of kSectionName is not correct.

Place the section name as a class member, not as a constant.